### ISSUE TITLE
nosetests fail due to missing tf_utils.py

### ISSUE THREAD

**Issue by mcflugen** [permamodel/permamodel#9]
Running `nosetests` on permamodel currently fails. That is,

``` bash
$ nosetests --with-doctest permamodel
```

The problem is on line 29 of `pixels.py` (`from tf_utils import TF_Print`), which raises the following `ImportError`:

``` python
ImportError: No module named tf_utils
```

Since `tf_utils.py` is not part of the permamodel package, I suggest removing the import and replacing the `TF_Print` calls with regular-old prints or with a permamodel-specific print statement.

**Comment by Elchin** [comment]
Its left from the old Scott's BMI code and should be removed, I think.

**Comment by sc0tts** [comment]
I like that Peckham is "old Scott"!

I'll replace TF_Print with print

-Scott (S)

**Comment by mdpiper** [comment]
Fixed by @sc0tts with d5bffd856aeda7b5c37ecde8d94b7492defd0c55.
